src/robusta/integrations/jira/sender.py

```diff
@@ -200,6 +209,11 @@ def send_finding_to_jira(
                     to_paragraph(f"🎬 {video_link.name}", [{"type": "link", "attrs": {"href": video_link.url}}])
                 )
 
+        # Add runbook_url to issue markdown if present
+        if finding.subject.annotations["runbook_url"]:
```

Comment:
This will throw KeyError in case there's no `runbook_url`
Might be better to use `finding.subject.annotations.get("runbook_url", None)`

Reply:
Done, thanks